srml/council/src/seats.rs

```diff
@@ -446,14 +490,24 @@ impl<T: Trait> Module<T> {
 		Ok(())
 	}
 
-	/// Remove a voter from the system. Trusts that Self::voters()[index] != voter.
-	fn remove_voter(voter: &T::AccountId, index: usize, mut voters: Vec<T::AccountId>) {
-		<Voters<T>>::put({ voters.swap_remove(index); voters });
+	/// Remove a voter at a specified index from the system.
+	///
+	/// Trusts that Self::voters()[index] != voter.
```

Comment:
```suggestion
	/// Trusts that Self::voters()[index] == voter.
```

Reply:
This comment was already there and at first glance, it seemed correct to me; and now that I think about it neither is correct. 

- When a `reap_inactive_voter` is happening => `Self::voters()[index] != voter`
- When a `retract_voter` is happening => `Self::voters()[index] == voter`

None seems wrong actually.